cardano/src/util/arbitrary.rs

```diff
@@ -75,20 +71,20 @@ impl Arbitrary for Wrapper<(hdwallet::XPrv, address::ExtendedAddr)> {
         let attributes = address::Attributes {
             derivation_path: None,
             stake_distribution: address::StakeDistribution::BootstrapEraDistr,
-            network_magic: *network_magic,
+            network_magic: network_magic.into_inner(),
         };
         let addr_type = address::AddrType::ATPubKey;
         let addr = address::HashedSpendingData::new(
             addr_type,
-            &address::SpendingData::PubKeyASD(xprv.public()),
+            &address::SpendingData::PubKeyASD(xprv.clone().into_inner().public()),
```

Comment:
It's more performant to have a by-ref accessor here and clone the inner object: `xprv.wotsit().clone()`